Cargo.toml

```diff
@@ -29,5 +29,5 @@ tokio = { version = "1.1", features = ["sync", "time"] }
 tokio-stream = { version = "0.1" }
 tracing = "0.1.13"
 
-near-indexer-primitives = ">=0.15.0,<0.16.0"
+near-indexer-primitives = "0.16.0"
```

Comment:
I believe we should either require `=0.16.0` or follow the previous pattern `>=0.16.0,<0.17.0`

We would use such soft dependency in the past before it punched us straight in the face.

Reply:
```suggestion
near-indexer-primitives = ">=0.16.0,<0.17.0"
```